xbmc/pvr/dialogs/GUIDialogPVRChannelManager.cpp

```diff
@@ -616,6 +692,14 @@ void CGUIDialogPVRChannelManager::Update()
     SET_CONTROL_LABELS(SPIN_EPGSOURCE_SELECTION, 0, &labels);
   }
 
+  m_clientsWithSettingsList = g_PVRClients->GetClientsSupportingChannelSettings(m_bIsRadio);
+  if (!m_clientsWithSettingsList.empty())
+    m_bAllowNewChannel = true;
+
+  CGUIButtonControl *btnShowNewChannel = (CGUIButtonControl*) GetControl(BUTTON_NEW_CHANNEL);
+  if (btnShowNewChannel)
+    btnShowNewChannel->SetVisible(m_bAllowNewChannel);
```

Comment:
Upss, has in the night a small difficult in my mind, searched for the wrong name of it and thought is not used anymore ;-). Becomes fixed :-D